spec/models/spree/user_spec.rb

```diff
@@ -189,4 +189,11 @@
       expect { user.destroy }.to raise_exception(Spree::User::DestroyWithOrdersError)
     end
   end
+
+  describe "#flipper_id" do
+    it "provides a unique id" do
+      user = Spree::User.new(id: 42)
+      expect(user.flipper_id).to eq "Spree::User;42"
```

Comment:
Good choice of ID 🐬

Reply:
This one is mandatory: https://github.com/jnunemaker/flipper/blob/master/docs/Gates.md#2-individual-actor ;)